cypress/integration/happy-path-with-accessibility.js

```diff
@@ -110,26 +109,62 @@ describe('Get help to retrain smoke test', function() {
   });
 
   it('should allow me to choose training options', function() {
+    cy.get('[id="training_english_skills"]').check()
+    cy.get('[id="training_math_skills"]').check()
     cy.contains('Continue').click();
     checkAccessibility();
   });
 
-  it('should allow me to choose computer skills training options', function() {
+  it('should allow me to choose job hunting advice', function() {
+    cy.get('[id="it_training_computer_skills"]').check()   
     cy.contains('Continue').click();
     checkAccessibility();
   });
 
-  it('should allow me to choose job hunting advice and navigate to action plan', function() {
+  it('Navigate to action plan', function() {
+    cy.get('[id="job_hunting_cv"]').check();
+    cy.get('[id="job_hunting_cover_letter"').check();
+    cy.get('[id="job_hunting_interviews"]').check(); 
     cy.contains('Continue').click();
     checkAccessibility();
   });
 
   it('should allow me to find jobs near me', function() {
     cy.contains('Show jobs near me').click();
     checkAccessibility();
-  })
+  });
+
+  it('should allow me to find an maths course', function() {
+    cy.go('back')
+    cy.contains('Find a maths course').click()
+    checkAccessibility();
+  });
+
+  it('should allow me to find an English course', function() {
+    cy.go('back')
+    cy.contains('Find an English course').click()
+    checkAccessibility();
+  });
+
+  it('should allow me to find local schemes and offers', function() {
+    cy.go('back')
+    cy.contains('Show me local offers').click()
+    checkAccessibility();
+  });
+
```

Comment:
could we add steps to navigate to the 3 new pages for cv, interview and cover letter options

Reply:
Will do this in separate PR.